ownCloudSDK/Drive/OCDrive.m

```diff
@@ -297,7 +298,7 @@ - (NSString *)description
 
 OCDriveType OCDriveTypePersonal = @"personal";
 OCDriveType OCDriveTypeVirtual = @"virtual";
-OCDriveType OCDriveTypeProject = @"project";
+OCDriveType OCDriveTypeProject = @"project"; // @"project";
```

Comment:
This change appears to be an accidental edit or leftover from development, but doesn't really change anything in the code. Please remove it.